localenv/README.md

```diff
@@ -124,12 +119,12 @@ Your local cloud nine rafiki instance will be peered automatically in this case
 The required services will be exposed externally using [tunnelmole](https://www.npmjs.com/package/tunnelmole) package.
 The exposed ports are 3000(open-payments), 3006(auth server), 3002(ilp connector).
 
-To use the postman collection examples follow the steps:
+To use the Bruno collection examples follow the steps:
 
 1. run `docker logs rafiki-cloud-nine-mock-ase-1`
 2. find the list of created wallet addresses
 3. copy the url of one of the wallet addresses
-4. set the url into `senderWalletAddress` postman variable in `Remote Environment`
+4. set the url as `senderWalletAddress` variable in the Bruno `Remote` environment
```

Comment:
should we mention the `clientPrivateKey` step in here?

Reply:
Couldn't we just pre-configure it according to the instructions instead of having these instructions? If we're suggesting to use the remote environment with a local wallet address then maybe we should just change https://ilp.rafiki.money/d99f0eee to http://localhost:3000/accounts/gfranklin? 

Or are we running bruno cli from some other context? Even still I imagine this use-case is basically the default.

I guess I'm also wondering why we want to point to the remote environment in the context of local development? I mean I get it in terms of integrating but for our development purposes why use remote?

I feel like these steps could basically be "load the collection from this path" and "select the local environment".

Reply:
Should we add an autopeering environment in the future?

Reply:
@sabineschaller that's a good idea :+1: